elide-model-config/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/validator/DynamicConfigValidator.java

```diff
@@ -185,21 +180,20 @@ private void flagDimensionsToOverride(Table table, Set<Table> parentTables) {
         Table current = table;
 
         while (current != null && current.getExtend() != null && !current.getExtend().equals("")) {
-            Table parent = getTableByName(current.getExtend().trim());
+            Table parent = getTableByName(this.elideTableConfig, current.getExtend().trim());
             parentTables.add(parent);
             current = parent;
         }
         return parentTables;
     }
 
-    private Table getTableByName(String tableName) {
-        Table tableByName = this.elideTableConfig.getTables().stream().filter(
-                table -> table.getName().equals(tableName)).findFirst().orElse(null);
-
-        if (tableByName != null) {
-            return tableByName;
-        }
-        throw new IllegalStateException("Table " + tableName + " is not defined in Dynamic Config.");
+    private static Table getTableByName(ElideTableConfig elideTableConfig, String tableName) {
+        return elideTableConfig.getTables()
```

Comment:
Seems like another function that could be moved to ElideTableConfig?

Reply:
updated.